### ISSUE TITLE
Bug: Scroll issue on the feed

### ISSUE THREAD

**Issue by pritipsingh** [open-sauced/app#2406]
### Describe the bug

I am not sure if this is how it's supposed to be but definitely not the best UX imo. When I am on the feed and I try to scroll through the page - I can't scroll the left and right sections. And my screen is at proper 100% zoom. Could be different for different screens. 

But ideally if the section is more than the height of the screen I should be able to scroll it as well. And it couldn’t be scrolled until I am at the bottom of the screen.

### Steps to reproduce

- Go to the main feed: https://app.opensauced.pizza/feed
- If the left and right section is not more than the height of the screen like this (increase the zoom of the screen - even though mine at 100% it's more than the height of the screen) :
<img width="1440" alt="ss of the feed image" src="https://github.com/open-sauced/app/assets/85965351/cfcd5bdf-b28b-49b9-8026-fa3308654eaf">

- Try to scroll over it. I want to click the button but I can't see what the button on the left section says. I can only see it if I am at the bottom of it.

Attaching a video below:

https://github.com/open-sauced/app/assets/85965351/0fb43ff3-e6a3-4b14-b356-b1d61d29d730

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by sourabpramanik** [comment]
@pritipsingh I have made some changes which will make all the three sections to scroll together like this 

![fix](https://github.com/open-sauced/app/assets/61370770/674698db-5da9-4d2d-854a-779431139e2c)
Also we can rather add individual scroll to each of the sections. Let me know what do you think.

Also I am not sure if this is a good UX as per the core/design team, so will wait for their review as well on this.

**Comment by pritipsingh** [comment]
I agree the left and right sections should have it's own scroll just like twitter. But waiting for the official design team's intervention is the best way forward in this!
